packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -380,6 +384,11 @@ const ReactTestUtils = {
 
   Simulate: null,
   SimulateNative: {},
+
+  act(callback: () => void) {
+    ReactDOM.unstable_batchedUpdates(callback);
+    ReactDOM.render(actElement, actContainerElement);
```

Comment:
I then we should make it return result of calling batched updates (which by itself passes the callback retvalue).

This lets you write 

```js
const inst = act(() => ReactDOM.render(...)))
```

without awkward separate declaration.